netmanager/src/views/layouts/common/Sidebar/Sidebar.js

```diff
@@ -117,7 +117,8 @@ const allMainPages = [
     nested: true,
     nestItems: [
       { title: 'Network Map', href: '/manager/map' },
-      { title: 'Network Statistics', href: '/manager/stats' }
+      { title: 'Network Statistics', href: '/manager/stats' },
+      { title: 'Site Activity Logs', href: '/manager/activities' }
```

Comment:
@Codebmk , @danielmarv , perhaps we could call it `Network Activity Logs`?
Just thought about it this morning. :)
Just to help maintain the flow of things from the rest of the item in sidebar popup.

Thanks!

Reply:
@Baalmart it is resolved